charts/all/medical-diagnosis/xray-init/templates/riskAssessment/kafkaservice.yaml

```diff
@@ -14,7 +14,7 @@ spec:
           revisionTimestamp: ''
     spec:
       containers:
-      - image: quay.io/rh-data-services/xraylab-risk-assessment:latest
+      - image: quay.io/hybridcloudpatterns/hardware-accelerated-risk-assessment:latest
```

Comment:
@day0hero can we perhaps drive this from `value-hub.yaml` as an application override?
Maybe we should make a copy of it... `values-amx.yaml`